cmd/zb/perf.go

```diff
@@ -147,11 +148,16 @@ type statsRecord struct {
 	statusCode int
 	isConnFail bool
 	isErr      bool
+	err        error
 }
 
 func updateStats(summary *statsSummary, record statsRecord) {
 	if record.isConnFail || record.isErr {
-		summary.errors++
+		summary.errorCount++
+	}
+
+	if record.err != nil {
+		summary.errors = append(summary.errors, record.err)
```

Comment:
If every request is an error, this could get very large?

Reply:
Maybe you want buckets of things and their counts?

Reply:
Right now we have counts.

The issue is we only have counts. I was getting client side errors when the client was making requests and I did not know what they were, as status code did not get to be checked and error messages were not shown in server logs. (With this patch I found them to be EOF errors, which were probably caused by other requests deleting the file while the server was reading it for the errored request)

I think we can improve by having a map of errors and counting duplicates.